extensions/interactions/rules.json

```diff
@@ -0,0 +1,186 @@
+{
+  "CodeRepl": {
```

Comment:
can extensions/interactions/CodeRepl contain rules.json? The same for other interactions, so that they are self-contained?